hack/test-end-to-end-docker.sh

```diff
@@ -36,7 +36,7 @@ function cleanup()
 	dump_container_logs
 
 	echo "[INFO] Dumping all resources to ${LOG_DIR}/export_all.json"
-	oc export all --all-namespaces --raw -o json --config=${ADMIN_KUBECONFIG} > ${LOG_DIR}/export_all.json
+	oc export all --all-namespaces --raw -o json --config=${ADMIN_KUBECONFIG-} > ${LOG_DIR}/export_all.json
```

Comment:
We prefer to use `${var:-default}` because it's rare that someone wants the set-but-null behavior from `${var-default}` and keeping that nuance in mind is hard.

Also, why do we expect this export to work if we *don't* have `$ADMIN_KUBECONFIG`? AFAICT, it is *always* set once a cluster has been successfully stood up. If one hasn't, would we expect `oc export` to work? If one has, would we expect the output of `oc export` to be useful without the admin credentials?